templates/config.hocon.tmpl

```diff
@@ -149,7 +149,7 @@
 
   "telemetry": {
     "disable": ${telemetry_disable}
-    "interval": 15 minutes
+    "interval": 1 minutes
```

Comment:
Reason to change this?  I think we generally want 15 minutes to avoid sending too much traffic volume back to ourselves?